sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -223,6 +223,44 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
     fromSequenceNumber?: Long;
 }
 
+// @public
+export interface ProcessErrorContext {
+    entityPath: string;
+    errorSource: "complete"
+    /**
+     * Covers errors that occur when the user (or autoComplete) abandons a message.
+     */
+     | "abandon"
+    /**
+     * Errors thrown from the user's `processMessage` callback passed to `subscribe`
+     */
+     | "userCallback"
+    /**
+     * Errors thrown when receiving messages.
+     */
+     | "receive"
+    /**
+     * Errors thrown when the user calls renewLock or when the internal lock renewer calls renewLock.
+     * Automatic lock renewal can be enabled via the `CreateReceiverOptions.maxAutoLockRenewalDurationInMs`
+     * property passed when calling `ServiceBusClient.createReceiver()`
+     */
+     | "renewLock"
+    /**
+     * Broadly covers a series of phases - the actual initialization that occurs when an operation
+     * is first started as well as errors that occur when reconnecting.
+     */
+     | "initialize"
```

Comment:
Why wouldnt this be part of `receive`?
cc @JoshLove-msft to see if the answer to the above question applies to .Net or not

Reply:
@JoshLove-msft  - also be interested to know how you report errors that have to do with connection reinitialization _after_ the processor has started (perhaps they just show up as 'receive' as well?)

Reply:
Yep, this would fall under "receive".

Reply:
Done. :)